test/unittest/unittest_inputsplit.cc

```diff
@@ -147,7 +147,7 @@ TEST(InputSplit, test_split_libsvm_distributed) {
 #ifdef DMLC_UNIT_TESTS_USE_CMAKE
 /* Don't run the following when CMake is not used */
 
-#include <build_config.h>
+#include "build_config.h"
```

Comment:
Why this change?

Reply:
There are two "build_config.h".
Just want to make it clearer that the one in the same directory is used.

Reply:
That makes sense. Can you change it to `"./build_config.h"` to make it even clearer?

Reply:
Done. :)